### ISSUE TITLE
Yank back killed text `ctrl+y` stopped working

### ISSUE THREAD

**Issue by rkdmen** [Microsoft/vscode#72630]
- VSCode Version: 1.33.1 ( Recent )
- OS Version: Mac 10.14.4

Steps to Reproduce:

1. `ctrl+k` to kill text.
2. `ctrl+y` to paste the kill text does not work.

Does this issue occur when all extensions are disabled?: Yes/No

Works on all other program except vscode.

**Comment by rebornix** [comment]
We didn't bind `ctrl+y` to paste content, please help confirm if it's contributed by an extension?

**Comment by rkdmen** [comment]
@rebornix Currently VSCode is only app on my mac that does not support `ctrl+y`. Something must’ve over ride that. It’s should work natively without binding.

**Comment by yjkogan** [comment]
I think this may have broken in Mojave specifically. I recently went from a work computer on Sierra to a personal computer that's running Mojave and noticed this. I'm just guessing at the moment though

**Comment by rkdmen** [comment]
@rebornix It would be really nice to get `yank` back working. It's the only app in mac that doesn't work.

**Comment by partap** [comment]
It's not the same as paste (`cmd+v`)...
`ctrl+k`, `ctrl+y` uses a different buffer than the clipboard.

Although, I suppose that's not a dealbreaker... As a workaround I could map `ctrl+k to shift+cmd+right_arrow, cut`, and then map `ctrl+y` to paste, just to help my poor muscle memory.

But it seems that vscode is actively overriding the system's `ctrl+y` behavior somehow, and it would be nice to be able to just "unmap" the key and make it work normally.

**Comment by lastobelus** [comment]
I don't believe this is specific to VSCode, in other Electron apps the behaviour is the same — kill (^k) works, but yank (^y) does not.

**Comment by rkdmen** [comment]
> @rebornix I don't believe this is specific to VSCode, in other Electron apps the behaviour is the same — kill (^k) works, but yank (^y) does not. (e.g.: Obsidian, Slack)

Spotify is Electron app, but it works (kill & yank)

**Comment by arobertn** [comment]
I'm not really sure if Electron on the Mac is using a standard Macos text widget that has these bindings implemented or not. However if you go to Code | Preferences | Keyboard Shortcuts and then search for Ctrl-k, it is in fact bound to "Delete All Right". If you remove that, Ctrl-k then does nothing in the editor.

So what is likely needed is a solution like @partap proposed, but to be delivered as part of the defaults. I am OK whether it uses the same clipboard (like Emacs does), or a separate one (like MacOS elsewhere does).

**Comment by notpeter** [comment]
I found an extension which does this and nothing more: [wdawson/vscode-bkr](https://marketplace.visualstudio.com/items?itemName=wdawson.better-kill-ring).

**Comment by partap** [comment]
@notpeter Yay! 😃
